packages/web/src/components/wip-ramp/commanders.tsx

```diff
@@ -74,7 +74,7 @@ export const RampComponent = () => {
     function toCapital(textIn: String) {
         return textIn.toUpperCase();
     }
-    if (Object.keys(myData).length === 0 && myData.constructor === Object) {
+    if (!myData) {
```

Comment:
Zmenili jsme to tak ze to vraci null, a null ==> false, takze zneguji vysledek podminky